packages/network/src/v2/ecsEventFromLog.ts

```diff
@@ -40,6 +40,16 @@ export const ecsEventFromLog = async (
     };
   }
 
+  if (name === "StoreEphemeralSetRecord") {
+    const value = await decodeStoreSetRecord(contract, tableId, args.key, args.data);
+    console.log("StoreEphemeralSetRecord:", { table: tableId.toString(), component, entity, value });
+    return {
+      ...ecsEvent,
+      ephemeral: true,
+      value,
+    };
+  }
+
```

Comment:
just another reminder
```suggestion
  if (name === "StoreEphemeralRecord") {
    const value = await decodeStoreSetRecord(contract, tableId, args.key, args.data);
    console.log("StoreEphemeralRecord:", { table: tableId.toString(), component, entity, value });
    return {
      ...ecsEvent,
      ephemeral: true,
      value,
    };
  }

```